src/server.js

```diff
@@ -0,0 +1,98 @@
+var fs = require('fs'),
+  dns = require('native-dns'),
+  dnsServer = dns.createServer(),
+  https = require('https'),
+  records = {
+    TXT: {},
+    A: {}
+  },
+  types = {
+    1: 'A',
+    16: 'TXT'
+  },
+  apiServer;
```

Comment:
Maybe move vars that do not come from a require to line 16. And use `const` when possible